instrumentation/active_job/test/instrumentation/active_job/patches/active_job_callbacks_test.rb

```diff
@@ -46,6 +46,12 @@
     end
   end
 
+  describe 'compatibility' do
```

Comment:
Would it be too much trouble to add test cases using non-keyword args and positional attributes?

Reply:
I added some additional tests, are there any other cases that you can think of that should be added?

Reply:
I think that covers it thanks!